verification/v0/phoneVerification.yaml

```diff
@@ -16,13 +14,13 @@ servers:
 tags:
   - name: "Phone Verification"
     description: |
-      Phone Verification service aims to provide information regarding the validity and the trustworthyness of a phone number.
+      Phone Verification service aims to provide information regarding the validity and trustworthiness of a phone number.
       This information comes in the form of a Risk Index about the phone number. The following risk classifications are applied
 
-      - **High:** The phone number considered as high risk number and customers should avoid contacting this number
-      - **Medium:** The phone number is generally considered legitimate number, but be countious when contacting this phone number
-      - **Low:** The phone number considered as legitimate number and customers should feel confident to contact this number
-  - name: metadata
+      - **High:** The phone number is considered a high-risk number and customers should avoid contacting this number
+      - **Medium:** The phone number is generally considered a legitimate number, but be cautious when contacting this phone number

```

Comment:
**Medium:** The phone number is generally considered a legitimate number, but customers should exercise caution when contacting this number for sensitive data or high-value transactions

Reply:
I've applied your suggestion: https://github.com/radekholy24/tyntec-api-collection/pull/226/commits/0dd01c2716c18bc2bdeffe896a36d3161e6bf2f8